Enterprise-CMCS/macpro-mdct-mfp
# Transition Benchmark Strategy - wp.json set up for UI

### Description
UI and mock data for Transition Benchmark Strategy - WP


https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/14514294/36e62b79-7fb5-47a1-9b85-bf9994bc32f8

<img width="1166" alt="Screenshot 2023-08-24 at 1 14 34 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/14514294/04b28f84-ab12-4b1a-8302-23ceac7d100e">


### Related ticket(s)
https://qmacbis.atlassian.net/jira/software/c/projects/MDCT/boards/232?modal=detail&selectedIssue=MDCT-2772
MDCT-2772

---
### How to test
1. Log in to MFP
2. Go go /wp/transition-benchmark-strategy
3. Should be able to toggle between /standard and /wp/transition-benchmark-strategy

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
